From c2a8c18dfded67095765f4421ee43ac3438b6ccd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Tue, 3 Oct 2023 14:57:31 +0200
Subject: [PATCH] downgrade pyzmq

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 5162326..61aaafd 100644
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@
         'cloudpickle>=2.2.1',
         'mpi4py>=3.1.4',
         'tqdm>=4.66.1',
-        'pyzmq>=25.1.1',
+        'pyzmq>=24.0.1',
     ],
     cmdclass=versioneer.get_cmdclass(),
 )
